consumer/junit5/src/main/kotlin/au/com/dius/pact/consumer/junit5/PactConsumerTestExt.kt

```diff
@@ -149,7 +149,8 @@ data class ProviderInfo @JvmOverloads constructor(
 
   companion object {
     fun fromAnnotation(annotation: PactTestFor): ProviderInfo =
-      ProviderInfo(annotation.providerName, annotation.hostInterface, annotation.port,
+      ProviderInfo(parseExpression(annotation.providerName, DataType.STRING)?.toString() ?: "default",
```

Comment:
default value should be `annotation.providerName`

Reply:
done